app/assets/javascripts/chessboard.js

```diff
@@ -0,0 +1,103 @@
+$(function() {
+    // draw the chess board
+    for (var y = 0; y <= 7; y++) {
+        $("#chess-board").append(
+          "<tr>" +
+            "<td data-x='0' data-y='" + y + "'></td>" +
+            "<td data-x='1' data-y='" + y + "'></td>" +
+            "<td data-x='2' data-y='" + y + "'></td>" +
+            "<td data-x='3' data-y='" + y + "'></td>" +
+            "<td data-x='4' data-y='" + y + "'></td>" +
+            "<td data-x='5' data-y='" + y + "'></td>" +
+            "<td data-x='6' data-y='" + y + "'></td>" +
+            "<td data-x='7' data-y='" + y + "'></td>" +
+          "</tr>");
+    }
+
+    // assign variables to initial positions based on
+    // their data attributes
+    // black piece variables
+    var blackRookLeft = 'td[data-x="0"][data-y="0"]';
```

Comment:
We tested and every time the page loads it picks up current positions of pieces in DB and draws them on the board accordingly.